test/helpdesk_tests/genbank_up_down_duplicate_test.py

```diff
@@ -0,0 +1,170 @@
+import os
+import time
+import unittest
+from configparser import ConfigParser
+
+from installed_clients.DataFileUtilClient import DataFileUtil
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from GenomeFileUtil.GenomeFileUtilServer import MethodContext
+from installed_clients.WorkspaceClient import Workspace as workspaceService
+
+
+class GenomeFileUtilTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        token = os.environ.get('KB_AUTH_TOKEN', None)
+        # WARNING: don't call any logging methods on the context object,
+        # it'll result in a NoneType error
+        cls.ctx = MethodContext(None)
+        cls.ctx.update({'token': token,
+                        'provenance': [
+                            {'service': 'GenomeFileUtil',
+                             'method': 'please_never_use_it_in_production',
+                             'method_params': []
+                             }],
+                        'authenticated': 1})
+        config_file = os.environ.get('KB_DEPLOYMENT_CONFIG', None)
+        cls.cfg = {}
+        config = ConfigParser()
+        config.read(config_file)
+        for nameval in config.items('GenomeFileUtil'):
+            cls.cfg[nameval[0]] = nameval[1]
+        cls.wsURL = cls.cfg['workspace-url']
+        cls.wsClient = workspaceService(cls.wsURL, token=token)
+        cls.serviceImpl = GenomeFileUtil(cls.cfg)
+        gbk_path = "data/Cyanidioschyzon/Cyanidioschyzon_merolae.ASM9120v1.30.dat"
```

Comment:
Is this actually a genbank file?

Reply:
yeah, its from Ensembl. I'm not sure why they had it listed as a '.dat' file